packages/terra-embedded-component/docs/README.md

```diff
@@ -13,9 +15,67 @@
 import React from 'react';
 import EmbeddedComponent from 'terra-embedded-component';
 
-<EmbeddedComponent {props...} />
+<EmbeddedComponent basePath="http://mycomponent.com/path" {props...} />
 ```
 
+## Building Embeddable Components
+
+EmbeddedComponent looks for a `manifest.json` at the `basePath` to identify the component dependencies.
+The JavaScript modules are exepcted to be in [AMD](https://en.wikipedia.org/wiki/Asynchronous_module_definition) format.
+In the future support for ES modules will be available. To make a React app work with EmbeddedComponent using webpack, do the following:
+
+### Create a manifest
+Generate a manifest which defines the assets dependencies. A [webpack manifest](https://webpack.js.org/guides/output-management/#the-manifest) can be generated automatically using the [webpack-manifest-plugin](https://github.com/danethurber/webpack-manifest-plugin).
+
+**Important**: `access-control-allow-origin: "*"` MUST be set on the response headers
+serving the manifest if the manifest is not hosted from the same domain as the embedding application.
+
+
+Example:
+
+```json
+{
+  "index.css": "index-ca5f6aac7b2f61572a96.css",
+  "index.js": "index.e5155c915e997dc22298.js",
+}
+```
+
+### Package the component as an AMD module
+In webpack set the following configuration for your `output` config:
+
+```js
+output: {
+  filename: '[name].[chunkhash].js',
+  path: path.join(__dirname, 'dist'),
+
+  // Required if the component is hosted from a path different than the embedding script file.
+  publicPath: 'http://componentdomain.com/',
```

Comment:
Is it possible to make this match with the example you have on line 18? I think that would be easier for consumers to see how to configure this portion.